src/main/java/com/wcc/platform/service/CmsService.java

```diff
@@ -0,0 +1,39 @@
+package com.wcc.platform.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.wcc.platform.domain.LeadershipMember;
+import com.wcc.platform.domain.pages.TeamPage;
+import com.wcc.platform.utils.FileUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CmsService {
+    private final ObjectMapper objectMapper;
+
+    @Autowired
+    public CmsService(ObjectMapper objectMapper) {
+        this.objectMapper = objectMapper;
+    }
+
+    public TeamPage getTeam() {
+        String teamPage = FileUtil.readFileAsString("teamPage.json");
+
+        try {
+            return objectMapper.readValue(teamPage, TeamPage.class);
```

Comment:
What is the need to read the .json file via inputstream and convert content to a string and then Map them to an object. It can be simplified by reading the Json file directly and pass to the object mapper. 
Something like this - ( Let me know if I am thinking something wrong)
         URL teamPageUrl = CmsService.class.getResource("/teamPage.json");
        try {
            if( teamPageUrl != null) {
                File file = Path.of(teamPageUrl.toURI()).toFile();
                return objectMapper.readValue(file, TeamPage.class);